src/test/java/de/tum/in/www1/artemis/assessment/ParticipantScoreIntegrationTest.java

```diff
@@ -213,6 +214,9 @@ void deleteParticipation_asInstructorOfCourse_shouldDeleteParticipation() throws
         }
         participations = studentParticipationRepository.findByExerciseIdAndStudentId(idOfIndividualTextExercise, idOfStudent1);
         assertThat(participations).isNotEmpty();
+
+        await().pollInterval(10, TimeUnit.MILLISECONDS).until(() -> participantScoreScheduleService.isIdle());
```

Comment:
By default, the pollInterval is set to 100, meaning that the condition is checked every 100 ms. With pollInterval(), we can decrease the time intervals between checks and in some cases, make the tests faster.

Btw. the pollDelay is also 100ms by default, meaning that the condition is only checked for the first time after 100ms have passed (This could make some tests significantly slower). Given the pollInterval, the pollDelay is set to the same value if not specified with pollDelay().

Reply:
Ok, makes sense. Thanks for the explanation!